### ISSUE TITLE
Gray border with transparent png: from 5.2.0

### ISSUE THREAD

**Issue by whitecolor** [itgalaxy/favicons#253]
I have original image:
![circle](https://user-images.githubusercontent.com/736697/50301673-c99cee80-04a9-11e9-979e-72e02e2d93a3.png)

From 5.2.0 the result image has gray border:
![android-chrome-144x144](https://user-images.githubusercontent.com/736697/50301692-d28dc000-04a9-11e9-8ac0-55dc181414dd.png)

In early version there is no border.

There also was an issue before:
https://github.com/itgalaxy/favicons/issues/123

**Comment by evilebottnawi** [comment]
@whitecolor feel free to send a PR with fix

**Comment by whitecolor** [comment]
Well, should be some stuff related to picture generation? Maybe `svg2png`? Need to see what changed between favicons 5.1.1 and 5.2.0.

**Comment by brunocodutra** [comment]
Right, it was certainly due to some dependency being updated, in which case this issue needs to be reported upstream once we find the culprit.

**Comment by egemon** [comment]
Sorry guys, thank u for doing such a good stuff. Do u think we can see a fix in some time?

**Comment by evilebottnawi** [comment]
PR welcome, it is open source and many peoples don't have enough time

**Comment by brunocodutra** [comment]
Actually a recent PR 254 replaced `svg2png` by `sharp`. 
@egemon do you have the means to test favicons@master and verify that the issue is gone?

**Comment by mikroware** [comment]
So that PR only changed the handling over svg images? I have this gray border issue with png images. So that PR would not have fixed my case. I will try to test this soon though and also export my image to svg to see if that works better.

**Comment by brunocodutra** [comment]
Good point, I don't see why we use both jimp and sharp, we should move to sharp everywhere

**Comment by niruix** [comment]
The same problem brought me here. 

And actually, I found that when `.svg` file is used instead of the `.png` one, the problem is gone.

I don't actually know any detail on the issue, but maybe take a look at the parameter used during PNG file processing?

**Comment by whitecolor** [comment]
@niruix thanks for pointing to `svg`.

**Comment by papiliond** [comment]
I still have this issue.
